plugins/azure-devops/src/index.ts

```diff
@@ -25,6 +25,10 @@ export {
 } from './plugin';
 
 export { AzurePullRequestsIcon } from './components/AzurePullRequestsIcon';
```

Comment:
Can we also export `AzurePipelinesIcon` while we are doing these exports for consistency sake?